test/e2e/resources/components/roSGNode.brs

```diff
@@ -35,6 +35,9 @@ sub main()
     print "field1 in node now is: " node1.getField("field1")      ' => hello
     print "field3 in node now is: " node1.getField("field3")      ' => false
 
+    print "field3 present? " node1.hasField("field3")
+    print "fieldほ present? " node1.hasField("fieldほ")
```

Comment:
Oh dang I've never thought about having non-ASCII characters in field names.  I'm glad that works! 🎉